xarray/backends/scipy_.py

```diff
@@ -222,6 +223,29 @@ def close(self):
         self._manager.close()
 
 
+def guess_can_open_scipy(store_spec):
+    # check byte header to determine file type
+    if isinstance(store_spec, bytes) or isinstance(store_spec, io.IOBase):
+        if isinstance(store_spec, bytes):
+            magic_number = store_spec[:8]
+        else:
+            if store_spec.tell() != 0:
+                raise ValueError(
+                    "file-like object read/write pointer not at zero "
```

Comment:
is that a user-facing error? If so, it might be better to replace "zero" with "the start of the file.".

Reply:
Yes, it is user facing an I like your suggestion.

I coped the message text from `_get_engine_from_magic_number`, but I can sync all the instances.